test/e2e/projects_test.go

```diff
@@ -64,7 +64,7 @@ var _ = Describe("Nutanix projects", func() {
 		dumpSpecResourcesAndCleanup(ctx, specName, bootstrapClusterProxy, artifactFolder, namespace, cancelWatches, clusterResources.Cluster, e2eConfig.GetIntervals, skipCleanup)
 	})
 
-	It("Create a cluster linked to non-existing project (should fail)", func() {
```

Comment:
can we use same one as in https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pull/118/files which is capx-feature-test

Reply:
Removed the change